lib/kamal/secrets/adapters/bitwarden.rb

```diff
@@ -29,6 +29,7 @@ def fetch_secrets(secrets, account:, session:)
           item_json = JSON.parse(item_json)
 
           if fields.any?
+            fields = item_json["fields"].pluck("name") if fields == [ "__ALL__" ]
```

Comment:
I think we can just allow the name to be passed without any fields and then return all the values - so something like this:

```ruby
          elsif (item_fields = item_json.dig("fields"))
            item_fields.each do |item_field|
              results["#{item}/#{item_field["name"]}"] = item_field["value"]
            end
          else
```

Then we can avoid the special `__ALL__` value and you can say:

```
SECRETS=$(kamal secrets fetch --adapter bitwarden --account email@example.com ITEM)
FIELD1=$(kamal secrets extract FIELD1)
```

Reply:
this is in conflict with the current code which just defaults to extracting a password from the item selected
currently `secrets fetch ITEM` returns a hash {"ITEM" => "password from item"} for login item, otherwise an error
another idea I had was to add it as an arg option `kamal secrets fetch --all-fields ITEM` not sure how feasible that is as it would impact the overall adapter API
another idea is to do make `secrets fetch ITEM` return all the fields as { "FIELD_1" => val, ... } if the item is not a Login item, that would not break compatibility
WDYT?

Reply:
Since I have received no feedback I have decided that the best way would be to do `kamal secrets fetch ITEM` in case the item is not a login item it will auto-fetch all the fields from that Item